bundles/org.codechecker.eclipse.plugin/src/org/codechecker/eclipse/plugin/report/ReportParser.java

```diff
@@ -42,9 +41,13 @@ public void processReports(){
             SearchList filteredReoports = new SearchList();
             for (String checker : reports.getCheckers()) 
                 for (ReportInfo rep : reports.getReportsFor(checker))
-                    if (Iterables.getLast(rep.getChildren().orNull().getItems()).getFile().equals(currentFileName)){
+                    for (BugPathItem bp : rep.getChildren().orNull().getItems())
+                        if (bp.getFile().equals(currentFileName)) {
+                            // if
+                            // (Iterables.getLast(rep.getChildren().orNull().getItems()).getFile().equals(currentFileName))
+                            // {
                         filteredReoports.addReports(ImmutableList.of(rep));
-                        for (SearchListener listener : listeners){
+                        for (SearchListener listener : listeners) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['for' has incorrect indentation level 24, expected level should be 28.](https://app.codacy.com/app/CodeChecker/CodeCheckerEclipsePlugin/pullRequest?prid=4641620)